plottable.d.ts

```diff
@@ -204,17 +208,14 @@ declare module Plottable {
              */
             has(key: K): boolean;
             /**
-             * Return an array of the values in the Map
-             *
-             * @return {V[]} The values in the store
-             */
-            values(): V[];
-            /**
-             * Return an array of keys in the Map.
+             * The forEach method executes the provided callback once for each key of the map which
+             * actually exist. It is not invoked for keys which have been deleted.
+             * However, it is executed for values which are present but have the value undefined.
```

Comment:
But it is not consistent with the rest of the class. All methods in this class have a description of what the method actually does. The entire class is a `shim`. Also when reading the documentation, I believe it makes more sense to have all information required there and not a redirect. Redirects are annoying and should only be used when the reader voluntarily needs more information.